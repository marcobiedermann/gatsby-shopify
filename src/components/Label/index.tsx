import React, { LabelHTMLAttributes } from 'react';
import { label } from './style.module.css';

export type LabelProps = LabelHTMLAttributes<HTMLLabelElement>;

function Label(props: LabelProps) {
  return <label className={label} {...props} />;
}

export default Label;
