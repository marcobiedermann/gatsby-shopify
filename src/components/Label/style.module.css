.label {
  color: #797475;
  display: block;
  font-size: 0.75rem;
}
