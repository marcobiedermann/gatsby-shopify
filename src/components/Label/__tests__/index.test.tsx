import { render } from '@testing-library/react';
import React from 'react';
import Label from '..';

describe('label component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Label>Label</Label>);

    expect(asFragment()).toMatchSnapshot();
  });
});
