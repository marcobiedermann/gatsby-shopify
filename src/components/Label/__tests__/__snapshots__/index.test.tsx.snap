// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`label component should render correctly 1`] = `
<label
  class="label"
>
  Label
</label>
`;
